### PR TITLE
chore(process-engine): Add Doc for HistoryCleanup Retries Config

### PR DESCRIPTION
**Pull request by psavidis**
relates to:  camunda/camunda-bpm-platform#2728

### CODE REVIEW

**Review by yanavasileva — COMMENTED** [review]
👍 Looks good. Let's move the property after `historyCleanupStrategy` in `History cleanup configuration parameters` section.
Hint: adding `id` to the property entry (see `historyCleanupStrategy`), creates anchor. Example: https://docs.camunda.org/manual/develop/reference/deployment-descriptors/tags/process-engine/#historyCleanupStrategy

**Comment by psavidis** [comment]
Property moved under `historyCleanupStrategy`

**Review by yanavasileva — APPROVED** [review]
👍

File: content/reference/deployment-descriptors/tags/process-engine.md [hunk 1]
```diff
@@ -1084,6 +1084,11 @@ The following is a list with the most commonly used process engine configuration
       <strong>Values:</strong> <code>removalTimeBased</code>, <code>endTimeBased</code>.
     </td>
   </tr>
+  <tr>
+    <td><code>historyCleanupDefaultNumberOfRetries</code></td>
+    <td>Integer</td>
+    <td>Specifies how many times a cleanup job will be executed before an incident is raised. Default value: <code>3</code></td>
```
Comment (tasso94) [inline content/reference/deployment-descriptors/tags/process-engine.md:1090]:
```suggestion
    <td>Specifies how often a cleanup job will be executed before an incident is raised. This property overrides the global <code>defaultNumberOfRetries</code> property which has a default value of <code>3</code></td>
```

**Review by tasso94 — CHANGES_REQUESTED** [review]

**Review by tasso94 — APPROVED** [review]
👍

**Pull request merged**